packages/plugin-cms-ui/src/modules/posts/components/sections/CustomFieldsSection.tsx

```
@@ -0,0 +1,100 @@
+type FinalProps = {
+  fieldsGroupsQuery: FieldsGroupsQueryResponse;
+} & Props;
+
+const CustomFieldsSection = (props: FinalProps) => {
+  const { post, fieldsGroupsQuery, loading, isDetail, collapseCallback } =
+    props;
+  const [customFieldsData, setCustomFieldsData] = React.useState<any[]>(
+    post.customFieldsData
+  );
+
+  if (fieldsGroupsQuery && fieldsGroupsQuery.loading) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)